data-migration/migrate-sharding-mysql-tidb-above-tb.md

```diff
@@ -0,0 +1,438 @@
+   ```
+
+4. After starting the migration task, you can check the progress by using either of the following methods:
+
+   - View progress via the keyword `progress` in the `grep` log. By default, the data refreshes every 5 minutes.
+   - View progress via the monitoring dashboard. For more information, see [TiDB Lightning Monitoring]( /tidb-lightning/monitor-tidb-lightning.md).
+
+After the migration finishes, TiDB Lightning will exit automatically. To make sure that the data is migrated successfully, check for `the whole procedure completed` among the last 5 lines.
+
+> **Note:**
+>
+> Whether the migration is successful or not, the last line will always show `tidb lightning exit`. It just means that TiDB Lightning quits normally, and does not guarantee that the task is completed successfully.
+
+If you encounter any problems during migration, see [TiDB Lightning FAQs](/tidb-lightning/tidb-lightning-faq.md).
+
+## Step 3. (Optional) Use DM to perform incremental replication
+
+To migrate the source database to TiDB based on the binlog from a specified position, you can use TiDB DM to perform incremental replication.
+
+### Add the data source
+
+Create a new `source1.yaml` file, and add the following content:
+
+{{< copyable "" >}}
+
+```yaml
+# Configuration.
+source-id: "mysql-01" # Must be unique.
+
+# Specifies whether DM-worker pulls binlogs with GTID (Global Transaction Identifier).
+# The prerequisite is that you have already enabled GTID in the upstream MySQL.
+# If the upstream database has enabled automatic switch of primary and standby, you must use the GTID mode.
+enable-gtid: true
+
+from:
+  host: "${host}"           # For example: 172.16.10.81
+  user: "root"
+  password: "${password}"   # Plaintext passwords are supported but not recommended. It is recommended that you use dmctl encrypt to encrypt plaintext passwords.
+  port: ${port}             # For example: 3306
+```
+
+Run the following command in a terminal. Use `tiup dmctl` to load the data source configuration into the DM cluster:
+
+{{< copyable "shell-regular" >}}
+
+```shell
+tiup dmctl --master-addr ${advertise-addr} operate-source create source1.yaml
+```
+
+The parameters are described as follows.
+
+|Parameter      | Description |
+|-              |-            |
+|--master-addr         | {advertise-addr} of any DM-master node in the cluster that dmctl connects to. For example: 172.16.10.71:8261|
+| operate-source create | Load data sources to DM clusters. |
+
+Repeat the above steps until all MySQL instances are added to the DM.
+
+### Create a replication task
+
+Edit the `task.yaml` file to configure the incremental replication mode and replication starting point for each data source.
+
+{{< copyable "" >}}
+
+```yaml
+name: task-test               # The name of the task. Should be globally unique.
+task-mode: incremental        # The mode of the task. "incremental" means only incremental data is migrated.
```

Comment:
Maybe here we can emphasize that we'll use 'incremental' here because full data migration has finished by using Dumpling and TiDB Lightning